sig-architecture/annual-report-2021.md

```diff
@@ -3,80 +3,140 @@
+   increased reliability, Stabilizing, version skews etc have helped the community
+     - ecosystem balance shifted from using beta APIs primarily to using stable APIs primarily (1.22+)
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
+   - KEP survey for enhancements
+   - Working with dependencies to cut down indirect go.sum entries.
+   - Updating API conventions by adding documentation
 
-   -
-   -
-   -
-
-3. KEP work in 2021 (1.x, 1.y, 1.z):
-
-<!--
-In future, this will be generated from kubernetes/enhancements kep.yaml files
-1. with SIG as owning-sig or in participating-sigs
-2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
--->
+3. KEP work in 2021 (1.21, 1.22, 1.23):
 
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - [2527 - Clarify meaning of status](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/2527-clarify-status-observations-vs-rbac/README.md) - 1.22
+     - [1693 - Warning API mechanism](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1693-warnings/README.md) - 1.22
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-   - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [2572 - Defining the Kubernetes release cadence](https://github.com/kubernetes/enhancements/blob/master/keps/sig-release/2572-release-cadence/) - 1.23
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - PRR we need to grow more reviewers and approvers
+   - API reviews has a pipeline set up with Jordan leading. Need more SIGs to identify additional folks to start learning
+   - Conformance, all the easy stuff is done. So we have harder stuff to review, so need additional folks with context to help make progress
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
+   - We care about regressions, backports, dependencies across release boundaries
+   - We care about the number of enhancements per release along with their timely graduation
    -
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - TODO: Riaan
```

Comment:
@Riaankl there's a TODO for you here.

Reply:
Fixed. added a url to https://github.com/kubernetes/community/blob/master/sig-architecture/CONTRIBUTING.md